src/Cli/dotnet/Commands/Test/MTP/Terminal/TerminalTestReporterOptions.cs

```diff
@@ -52,9 +50,4 @@ internal sealed class TerminalTestReporterOptions
     /// Setting <see cref="UseAnsi"/> to false will disable this option.
     /// </summary>
     public bool UseCIAnsi { get; init; }
-
-    /// <summary>
-    /// Gets a value indicating whether we should force ANSI escape codes. When true the ANSI is used without auto-detecting capabilities of the console. This is needed only for testing.
-    /// </summary>
-    internal /* for testing */ bool? ForceAnsi { get; init; }
```

Comment:
how are we testing Ansi integration now?

Reply:
We never tested it in sdk repo.
This was copy from testfx repo.

Reply:
In the long term should we be testing the ansi functionality?

Reply:
Up to @nohwnd :)
So far it's dead code. It is easy to re-introduce it when needed in future.